packages/@o3r/design/builders/generate-jsonschema/index.ts

```diff
@@ -24,6 +24,7 @@ import type {
 } from './schema';
 
 const createBuilderWithMetricsIfInstalled: BuilderWrapper = (builderFn) => async (opts, ctx) => {
+  // eslint-disable-next-line unicorn/consistent-function-scoping -- keep for readability
```

Comment:
Could be extracted like in `packages/@o3r/design/builders/generate-css/index.ts`